projects/aca-content/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "aca-content",
```

Comment:
I think this file is not needed, as the library is being mapped in tsconfig

Reply:
We are going to make a npm package out of it, so I believe that we are going to need it